src/libAtomVM/atomshashtable.c

```diff
@@ -41,23 +41,10 @@
 struct HNode
 {
     struct HNode *next;
-    AtomString key;
+    AtomIndex key;
```

Comment:
So this turn to be an integer -> integer map. What about keeping just `valueshashtable` ?

Reply:
Thanks. Indeed, with a separate commit, this usage was moved to valueshashtable. Also, because atomhashtable was no longer referenced, it was removed.